src/effects/builtin/whitenoiseeffect.h

```diff
@@ -0,0 +1,48 @@
+#ifndef NOISEEFFECT_H
+#define NOISEEFFECT_H
+
+#include <QMap>
+
+#include "effects/effectprocessor.h"
+#include "engine/effects/engineeffect.h"
+#include "engine/effects/engineeffectparameter.h"
+#include "engine/filters/enginefilterpansingle.h"
+#include "util/class.h"
+#include "util/defs.h"
+#include "util/sample.h"
+#include "util/types.h"
+
+class NoiseGroupState final : public EffectState {
+  public:
+    NoiseGroupState(const mixxx::EngineParameters& bufferParameters)
+      : EffectState(bufferParameters),
+        m_framesSinceClickStart(0) {
+    }
+    ~NoiseGroupState() {
+    }
+
+    unsigned int m_framesSinceClickStart;
```

Comment:
What is this used for?

Reply:
This effect is forked from metronome and I forgot it, will remove it